aplib.net-demo/Assets/Scripts/Player/Movement.cs

```diff
@@ -93,10 +93,11 @@ private void MovePlayer()
             {
                 _rigidbody.AddForce(_maxSpeed * _acceleration * Time.fixedDeltaTime * _horizontalVelocity.normalized);
             }
+            
         }
         else
         {
-            _rigidbody.drag = 0;
+            _rigidbody.drag = 0.2f;
```

Comment:
If you're assigning this a value with meaning, you should define this somewhere else. I'd add an _airDrag field